src/AccountSelector.js

```diff
@@ -66,13 +67,15 @@ function Main (props) {
                 Polkadot JS Extension
               </a>
             </span>
-          ) : null}
-          <Icon
-            name='users'
-            size='large'
-            circular
-            color={accountSelected ? 'green' : 'red'}
-          />
+            : null }
+          <CopyToClipboard text={accountSelected}>
+            <Icon
+              name='users'
+              size='large'
+              circular
+              color={accountSelected ? 'green' : 'red'}
+            />
```

Comment:
```suggestion
            <Button
              basic
              circular
              size='large'
              icon='users'
              color={accountSelected ? 'green' : 'red'}
            >
            </Button>
```

Reply:
Looks better if it is a button because it changes the cursor and becomes clickable :)